File: services/device/src/main/java/com/example/iter/device/dto/response/EquipmentStatusResponse.java
package com.example.iter.device.dto.response;

import com.example.iter.device.domain.entity.EquipmentStatus;

import java.time.LocalDateTime;

public record EquipmentStatusResponse(
        Long equipmentId,
        EquipmentStatus status,
        LocalDateTime updatedAt
) {
}

File: services/device/src/main/kotlin/com/example/iter/device/dto/response/EquipmentStatusResponse.kt
package com.example.iter.device.dto.response

import com.example.iter.device.domain.entity.EquipmentStatus

import java.time.LocalDateTime

data class EquipmentStatusResponse(
    val equipmentId: Long,
    val status: EquipmentStatus,
    val updatedAt: LocalDateTime?,
)
